<?xml version="1.0"?>
<launch>
  
  <node pkg="object_detection_gl_ros" type="object_detection_gl_ros_node" name="object_detection_gl_ros_node" output="screen">
        <!-- General Params -->
        <param name="serial_port_name" type="string" value="/dev/ttyUSB0"/>
        <param name="frame_id" type="string" value="obj"/>
        <param name="pub_topicname_lidar" type="string" value="raw_lidar"/>

        <!-- Obj Params -->
        <param name="min_point_num" type="int" value="3"/>
        <param name="dist_thr" type="double" value="0.5"/>
        <param name="max_mean_point" type="int" value="10"/>
        <param name="max_traj_point" type="int" value="100"/>
        
        <!-- Node Params -->
        <param name="max_ref_count" type="int" value="40"/>
        <param name="pub_topicname_marker" type="string" value="marker_array"/>
        <param name="area_on" type="bool" value="true"/>
  </node>

  <node pkg="object_detection_gl_ros" type="text_subscriber" name="text_subscriber">
    
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find object_detection_gl_ros)/rviz/config.rviz" />

</launch>
